thinc/model.py

```diff
@@ -304,12 +306,12 @@ def finish_update(self, optimizer: Optimizer) -> None:
                 if node.has_grad(name):
                     param = node.get_param(name)
                     grad = node.get_grad(name)
-                    if hasattr(optimizer, 'ops'):
-                        param = optimizer.ops.asarray(param)        # type: ignore
-                        grad = optimizer.ops.asarray(grad)          # type: ignore
+                    if hasattr(optimizer, "ops"):
```

Comment:
is this check still useful ?

Reply:
It's not, no. We should remove that code